### PR TITLE
✨ landing curated list

### PR DESCRIPTION
**Pull request by roiLeo**
⚠️ DRAFT ⚠️ 

I've commented both component (LatestSales & NewestList) because it's freezing the screen and I'm still looking for a fix

### PR type

- [x] Feature

### What's new?

- [x] PR closes nothing?
- [x] New component CuratedList that will show "Trending" collection from a given list of Id (in `curatedCollection`)

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb)

### Screenshot

Screenshot 2022-02-16 at 16-47-28 KodaDot - Kusama NFT Market Explorer

### CODE REVIEW

**Comment by yangwao** [comment]
ah I see.. could be that we don't cache collections images? @vikiival 
Like substra knights took a while to load tho.

https://user-images.githubusercontent.com/5887929/154311495-e7e651a8-51ff-4a43-9088-04516c28cff7.mov

**Comment by roiLeo** [comment]
definitely not a problem with the query or cache system as when I replace it with a basic carousel, my screenmy screen always freezes and I can't play with the indicator (change slide)
Screenshot 2022-02-17 at 09-34-42 KodaDot - Kusama NFT Market Explorer

**Comment by vikiival** [comment]
> definitely not a problem with the query or cache system as when I replace it with a basic carousel, my screenmy screen always freezes and I can't play with the indicator (change slide)

can you rephrase that please 🥺  ?

**Comment by roiLeo** [comment]
> > definitely not a problem with the query or cache system as when I replace it with a basic carousel, my screenmy screen always freezes and I can't play with the indicator (change slide)
> 
> can you rephrase that please 🥺 ?

When I put the 3 carousel on the page, it becomes impossible to change slides in the first one.
I did a test by removing the images in the first slider to see if it came from the query or the size of the images but nothing changed, I still couldn't change the slide in the first slider.

**Comment by yangwao** [comment]
> When I put the 3 carousel on the page, it becomes impossible to change slides in the first one.

How impossible? Can you video it? For me it worked.

**Comment by yangwao** [comment]
oh I was totally brainless and forgot that two other comments are commented 
Anyway works seamless at my side 👀



https://user-images.githubusercontent.com/5887929/154796747-b63cc051-926f-4cae-8ac7-5e6ddddb9bdb.mov

**Comment by roiLeo** [comment]
continue in #2412
